core/tests/protractor/historyTab.js

```diff
@@ -98,7 +98,7 @@ describe('Exploration history', function() {
         highlighted: true
       },
       12: {
-        text: '        x: 6.0',
+        text: '        x: 6',
```

Comment:
The real reason is: because the e2e test failed otherwise.

However, I looked at the original code, and my question is actually more along the lines of: why was "6.0" even needed in the first place? So I figured the original behavior was incorrect and that it was fine to make this change.